scripts/hyperkit.js

```diff
@@ -0,0 +1,20 @@
+// This downloads hyperkit, builds it, and puts the binary in the right place.
+
+const fs = require('fs');
+const os = require('os');
+const { spawn, spawnSync } = require('child_process');
+
+fs.mkdirSync("./resources/" + os.platform(), { recursive: true });
+
+// The version of hyperkit to build
+let ver = "v0.20210107";
+
+// TODO: rewrite to remove callbackitis
+spawn('git', ['clone', '--depth', '1', '--branch', ver, "https://github.com/moby/hyperkit.git"], { cwd: '/tmp/' }).on('exit', () => {
+  spawn('make', [], { cwd: '/tmp/hyperkit/' }).on('exit', () => {
+    spawn('cp', ['-f', '/tmp/hyperkit/build/hyperkit', process.cwd() + '/resources/' + os.platform() + '/hyperkit']).on('exit', () => {
```

Comment:
Should probably use [`fs.copyFile()`](https://nodejs.org/api/fs.html#fs_fs_copyfile_src_dest_mode_callback).

Reply:
Good idea, done.